src/runtime/opengl.cpp

```diff
@@ -228,11 +228,10 @@ WEAK char *strndup(const char *s, size_t n) {
 }
 
 WEAK GLuint get_texture_id(buffer_t *buf) {
-    if (buf->dev == 0) {
-        return 0;
-    } else {
-        return halide_get_device_handle(buf->dev) & 0xffffffff;
-    }
+    uint64_t dev = buf->dev;
```

Comment:
Does this make the default behavior with a null dev ptr rendering to the frame buffer? Is this intentional and considered a good idea?